react-components/stories/ModelsStyling.stories.tsx

```diff
@@ -91,7 +90,6 @@ export const Main: Story = {
       <RevealContainer
         sdk={sdk}
         color={new Color(0x4a4a4a)}
-        uiElements={<ReactQueryDevtools initialIsOpen={false} />}
```

Comment:
I think we should add `ReactQueryDevtools` to most stories that involve queries in some way.